lib/ReactViews/StandardUserInterface/StandardUserInterface.tsx

```diff
@@ -217,208 +147,173 @@ const StandardUserInterface = observer<React.FC<StandardUserInterfaceProps>>(
       !props.viewState.explorerPanelIsVisible &&
       !props.viewState.storyBuilderShown;
     return (
-      <ThemeProvider theme={mergedTheme}>
-        <GlobalTerriaStyles />
-        <TourPortal viewState={props.viewState} />
-        <CollapsedNavigation viewState={props.viewState} />
-        <SatelliteHelpPrompt viewState={props.viewState} />
-        <Medium>
-          <SelectableDimensionWorkflow viewState={props.viewState} />
-        </Medium>
-        <div className={Styles.storyWrapper}>
-          {!props.viewState.disclaimerVisible && (
-            <WelcomeMessage viewState={props.viewState} />
-          )}
-          <div
-            className={Styles.uiRoot}
-            css={`
-              ${props.viewState.disclaimerVisible && `filter: blur(10px);`}
-            `}
-            onDragOver={handleDragOver}
-          >
+      <ViewStateProvider viewState={props.viewState}>
+        <ThemeProvider theme={mergedTheme}>
```

Comment:
I wonder if we should break out providers into their own file. Then we'll feel more free to add more providers and it won't pollute StandardUserInterface.

Reply:
Up to here with my review.

Reply:
done!